tmcprototype/SdpSubarrayLeafNode/setup.py

```diff
@@ -26,36 +26,35 @@
 
 pack = ['SdpSubarrayLeafNode']
 
-setup(
-    name=name,
-    version=version,
-    description='A Leaf control node for DishMaster.',
-    packages=pack,
-    include_package_data=True,
-    test_suite="test",
-    entry_points={'console_scripts':['SdpSubarrayLeafNode = SdpSubarrayLeafNode:main']},
-    author='jyotin.ska',
-    author_email='jyotin.ska at gmail.com',
-    license='BSD-3-Clause',
-    long_description=long_description,
-    url='www.tango-controls.org',
-    platforms="All Platforms",
-    install_requires=['pytango==9.2.5', 'mock', 'katpoint'],
-    #test_suite='test',
-    setup_requires=[
-        # dependency for `python setup.py test`
-        'pytest-runner',
-        # dependencies for `python setup.py build_sphinx`
-        'sphinx',
-        'recommonmark'
-    ],
-    tests_require=[
-        'pytest',
-        'pytest-cov',
-        'pytest-json-report',
-        'pycodestyle',
-    ],
-    extras_require={
-        'dev':  ['prospector[with_pyroma]', 'yapf', 'isort']
-    }
-)
+setup(name=name,
+      version=version,
+      description='A Leaf control node for DishMaster.',
+      packages=pack,
+      include_package_data=True,
+      test_suite="test",
+      entry_points={'console_scripts': ['SdpSubarrayLeafNode = SdpSubarrayLeafNode:main']},
+      author='jyotin.ska',
+      author_email='jyotin.ska at gmail.com',
```

Comment:
can replace with the actual email id.. :rofl:

Reply:
Done.